lib/api/completeMultipartUpload.js

```diff
@@ -457,7 +457,7 @@ function completeMultipartUpload(authInfo, request, log, callback) {
                 //   the previous MPU completion attempt
                 //
                 if (!locationKeysHaveChanged(objMD.location, dataLocations)) {
-                    log.info('MPU complete request replay detected', {
+                    log.debug('MPU complete request replay detected', {
```

Comment:
should not happen often, may be best to keep in INFO logs? or is this really so common (and safe) that this should only be in `debug`?

Reply:
Yes, why not, I'm reverting this